src/lib/handlebars-converter/handlebars-helpers.spec.js

```diff
@@ -222,7 +258,7 @@ describe('Handlebars helpers', function () {
         var currentString = getHelper('now').func();
         var after = new Date();
 
-        var current = helperUtils.getDate(currentString);
+        var current = new Date(helperUtils.getDateTime(currentString));
```

Comment:
why is this change needed?

Reply:
because the type of data the getDateTime return is a string, I use toJSON() in return.